custom-recipes/api-connect/recipe.json

```diff
@@ -237,6 +237,19 @@
             "type": "SEPARATOR",
             "label": "Advanced"
         },
+        {
+            "name": "behaviour_when_error",
+            "label": "Error behaviour",
+            "description": "Decide how the recipe should react when an input line results in an error",
+            "type": "SELECT",
+            "defaultValue": "keep-error-column",
```

Comment:
🥜 nitpick: why have `keep-error-column` as default when `add-error-column` is set everywhere else ?

Reply:
This is the way to enforced the new preferred behavior on new flows while keeping the same behavior on preexisting flows. 
This selector did not exist in the previous version, so is not saved in existing flows that are already using the plugin. As a result, the behavior of existing flows will not be changed, because in absence of the `behaviour_when_error` parameter, `add-error-column`(the previous behavior) is the default defined in the code. When a new recipe is created, the new behavior is the one set by default in the UI, the preferred and more stable "keep-error-column"

Reply:
Ok, no problem them 👌